src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -290,7 +290,10 @@ private static bool IsAfterDot(ITextSnapshot snapshot, ITrackingSpan applicableT
             if (bestItem != null)
             {
                 selectedItemIndex = itemsInList.IndexOf(i => Equals(i.FilterResult.CompletionItem, bestItem));
-                if (selectedItemIndex > -1 && bestItem != null && matchingItems.Length == 1 && filterText.Length > 0)
+                if (selectedItemIndex > -1 &&
+                    bestItem != null &&
+                    matchingItems.Where(r => !r.DisplayText.StartsWith("★")).Count() == 1 &&
+                    filterText.Length > 0)
                 {
                     uniqueItem = highlightedList[selectedItemIndex].CompletionItem;
```

Comment:
A bit concerned that we aren't explicitly grabbing the unique non-★ item to report as the `uniqueItem`. Will it work out that the ★'d item always wins? Is there any way this could go wrong? Also a detailed comment would be nice.

Reply:
@dpoeschl , thank you! I have updated the PR.